test.js

```diff
@@ -1,4 +1,4 @@
-/* global describe, test, beforeEach, afterEach, expect */
+// global describe, test, beforeEach, afterEach, expect
```

Comment:
I don't think these are necessary if you use the jest plugin

Reply:
ESLint was earlier giving no-redeclare errors earlier: `'describe' is already defined as a built-in global variable    no-redeclare`. While switching to a single line comment seemed to resolve that, I'm not really sure if that comment is needed anymore so I think removing the comment altogether is a better approach. Please let me know it needs to be reverted